interfaces/Portalicious/src/app/components/query-table/query-table.component.ts

```diff
@@ -484,9 +489,43 @@ export class QueryTableComponent<TData extends { id: PropertyKey }, TContext> {
    */
   visibleColumns = model<QueryTableColumn<TData>[]>([]);
 
-  resetColumnVisibility() {
+  private getStoredColumns(stateKey: string): null | QueryTableColumn<TData>[] {
+    const storedColumns = localStorage.getItem(stateKey);
+    if (!storedColumns) return null;
+
+    return JSON.parse(storedColumns) as QueryTableColumn<TData>[];
+  }
+
+  private getMatchingColumns(
+    storedColumns: QueryTableColumn<TData>[],
+  ): QueryTableColumn<TData>[] {
+    return storedColumns
+      .map((column) => this.columns().find((c) => c.field === column.field))
+      .filter(Boolean) as QueryTableColumn<TData>[];
+  }
+
+  private getDefaultColumns(): QueryTableColumn<TData>[] {
```

Comment:
```suggestion
  private defaultColumns = computed<QueryTableColumn<TData>[]>(() => {
```

Not a necessary change but this feels like it would be more in line with how we normally name / treat computed properties like this one.

Reply:
I think a computed only has benefits when it's used in the html. So that's why I chose it to be a regular method. Or am I missing something here?

Reply:
Yeah, this wouldn't necessarily have _benefits_ from a technical / performance aspect or anything similar. It's more about avoiding introducing a "getter" pattern for something that can be a `computed`, and this being picked up elsewhere

Reply:
I'll let you decide if you wanna change or if you think I'm overthinking things (which I might be 😄 )